packages/pharos/src/components/image-card/pharos-image-card.scss

```
@@ -137,6 +149,13 @@ slot[name='metadata']::slotted(strong) {
   color: var(--pharos-color-text-20);
 }
 
+:host([variant='promotional']) .card__metadata {
+  @include mixins.font-base(
+    $font-size: var(--pharos-font-size-base),
+    $line-height: var(--pharos-line-height-medium)
+  );
```

```suggestion
  @include mixins.font-base;
```
Those values are the defaults

---

Ah thanks—had started with different values and after changing them didn't realize.